src/components/Button/README.md

```diff
@@ -181,16 +192,6 @@ of your choice and display label once you know there is enough room for it.
     labelVisibility="xl"
     beforeLabel={<rui-icon icon="pencil" />}
   />
-  <Button
-    label="Label visible from x2l up"
-    labelVisibility="x2l"
-    beforeLabel={<rui-icon icon="pencil" />}
-  />
-  <Button
-    label="Label visible from x3l up"
-    labelVisibility="x3l"
-    beforeLabel={<rui-icon icon="pencil" />}
-  />
```

Comment:
Why removing these? You think it's too much to show everything? (I'm not against your change, after all.)

Reply:
> You think it's too much to show everything?

Yes, that is the reason. I think the functionality is sufficiently shown without them and too much code makes for worse readability.

I can put it back, I have no strong feelings about it.